Robust_Lab.qmd

```diff
@@ -41,85 +47,295 @@ Use the `stars` data in `robustbase`. This data looks at the relationship betwee
+
+```
+
 e\. Compare the observed mean difference to the NHD (is *p* \< .05?)
 
+Yes, *p* \< .05, as the observed mean difference does not overlap with the central 95% of the null distribution.
+
 6.  Suppose a replication experiment was conducted to further examine the interaction effect between driving difficulty and conversation difficulty on driving errors in a driving simulator. In the replication, the researchers administered the same three levels of conversation difficulty; (1) control, (2) easy, (3) difficult (C, E, D) but assume that they added a third level of driving difficulty; (1) low, (2) moderate, (3) difficult (L, M, D). Assume the design was completely between subjects and conduct a factorial ANOVA to test the main effects of conversation and driving difficulty as well as the interaction effect. The DV is the number of errors committed in the driving simulator.
 
     ```{r}
     library(tidyverse)
-    fac_data<-read_csv("https://raw.githubusercontent.com/jgeller112/psy503-psych_stats/master/static/assignment/data/fact_final.csv")
-
-
+    fac_data<-read_csv("https://raw.githubusercontent.com/jgeller112/psy503-psych_stats/master/static/assignment/data/fact_final.csv", show_col_types=FALSE) %>%
+      select(convo,drive,errors)
     ```
 
     a\. Run a permutation test (ANOVA)
 
+    ```{r}
+    set.seed(21)
+    anova_results <- permuco::aovperm(errors ~ convo*drive, data = fac_data, np=10000)
+    kable(anova_results$table)
+    ```
+
     b\. How would you follow-up significant effects in this context?
 
+    ```{r}
+    normalaov <- aov(errors ~ convo*drive, data = fac_data)
+      tukey <- TukeyHSD(normalaov)
+      kable(tukey$convo, caption="Tukey test - Conversation difficulty")
+      kable(tukey$drive, caption = "Tukey test - Driving difficulty")
+
+      fac_data$drive <- factor(fac_data$drive, levels = c("L","M","D")) 
+      fac_data$convo <- factor(fac_data$convo, levels = c("C","E","D"))
+       ggplot(fac_data, aes(x=drive, y = errors, group=convo, color=convo)) +
+         labs(x="Driving Difficulty", y="Errors", color="Conversation Difficulty") +
+         stat_summary(fun.y=mean, geom="point") +
+         stat_summary(fun.y=mean, geom="line") +
+      scale_x_discrete(labels=c('Low', 'Moderate','Difficult')) +
+         scale_color_discrete(labels=c('Control','Easy','Difficult'))
+    ```
+
+The ANOVA results indicate that conversation difficulty and driving difficulty both have significant effects on driving errors, but the interaction effect is not significant. I would carry out a post-hoc Tukey test, although I couldn't find a package that would do this for aovperm. Results from the normal (no permutations) ANOVA indicate that all differences between groups, both for driving difficulty and conversation difficulty, are significant. From the graph above, we can see that the number of driving errors is greater when the conversation difficulty is higher, and when the driving difficulty is higher.
+
```

Comment:
When nonparametric stats are used, I would not follow up with parametric stats. What I would do is take pairwise means/differences and bootstrap them and then use confidence intervals as my significance test. You could use permutation tests, but that just tells you if there is an effect, not how precise is the effect.